_posts/2020-08-28-accesibility-in-mind.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: How designing with accessibility in mind benefits your web app
+author: jalef
+excerpt: >
+  Accessibility is a requirement for plenty of projects nowdays - especially the ones developed for the public sector. But why implement accessibility only when asked, istead of starting designing and developing with accessibility in mind? 
+tags:
+ - accessibility
+ - frontend
+ - UI/UX
+---
+
+The W3C has published the Web Content Accessibility Guidelines (WCAG) [document](https://www.w3.org/WAI/standards-guidelines/wcag/) in an effort to make the Web more accessible to people with disabilities. Our disabled fellow netizens may be a minority demographic, but the WCAG is relevant and useful to all Web Designers and Developers. In fact, the only fault in the document is its stated purpose, which might mislead you into thinking it won't help improve your flashy web app. Nothing could be farther from the truth! Let's examine why.
```

Comment:
"Web Designers and Developers" -> "web designers and developers"

"farther from the truth" -> "further from the truth"

Reply:
done